### PR TITLE
Add a library in the repository

### PR DESCRIPTION
**Pull request by pocke**

### CODE REVIEW

File: lib/foo.go [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package lib
+
+func F(x int) int {
```
Comment (ghost) [inline lib/foo.go:3]: __[golint]__
* [golint] exported function F should have comment or be unexported

Comment (ghost) [inline lib/foo.go:3]: __[gometalinter]__
* [golint] exported function F should have comment or be unexported

Comment (sweep3092) [inline lib/foo.go:3]: __[staging][gometalinter]__
* [golint] exported function F should have comment or be unexported

Comment (sweep3092) [inline lib/foo.go:3]: __[staging][golint]__
* [golint] exported function F should have comment or be unexported

**Pull request closed without merging**